tests/scaffoldings/settings/custom/scripts/test_smtp_settings.py

```diff
@@ -11,3 +11,4 @@
 assert config.get("smtp_server") == "mailhog"
 assert config.get("smtp_ssl") is False
 assert config.get("smtp_user") is False
+assert config.get("dbfilter") == ".*"
```

Comment:
You nailed it!

Could you please rename the file? It's not only smtp anymore.

Remember to rename it in the test too.